pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -239,9 +239,75 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Monitor the cloud-credential-operator-config configmap to check if operator is re-enabled.
+	// Check if attempts were made to delete the credentials requests when the operator was disabled.
+	// We have to have to reconcile all those credential requests.
+	configmapMapFn := handler.ToRequestsFunc(
+		func(a handler.MapObject) []reconcile.Request {
+			// Iterate all CredentialsRequests to determine if any of them have DeletionTimestamp set
+			log.Debug("checking for credentials requests marked for cleanup when operator was disabled")
+			crs := &minterv1.CredentialsRequestList{}
+			mgr.GetClient().List(context.TODO(), crs)
+			requests := []reconcile.Request{}
+			for _, cr := range crs.Items {
+				if cr.DeletionTimestamp != nil {
+					log.WithFields(log.Fields{
+						"cr": fmt.Sprintf("%s/%s", cr.Namespace, cr.Name),
+					}).Info("found credentials request marked for cleanup")
```

Comment:
Per above just skip checking if deleted, and log at Info that we're requeuing all cred requests because the config changed.